color-theme-sanityinc-tomorrow.el

```diff
@@ -575,6 +575,9 @@ names to which it refers are bound."
       (show-paren-match (:background ,purple :foreground ,background))
       (show-paren-mismatch (:background ,red :foreground ,background))
 
+      ;; perspective
+      (persp-selected-face (:weight bold :foreground ,yellow))
```

Comment:
These new sections should be added in positions that preserve alphabetical order, please.